tests/ZiraLink.Tests/Tools/TestTools.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Microsoft.EntityFrameworkCore;
+
+using Moq;
+
+using ZiraLink.Api.Application;
+using ZiraLink.Domain;
+using ZiraLink.Domain.Enums;
+
+namespace ZiraLink.Tests.Tools
+{
+    public class TestTools
+    {
+
+        public static DbContextOptions<AppDbContext>? contextOptions;
```

Comment:
Better to put a underscore behind the variable name

Reply:
I forgot. 😊